internal/version

```diff
@@ -0,0 +1 @@
+v0.5.0
```

Comment:
sorry - maybe I wasn't clear enough with my previous comment. What I was meaning is that we should have the golang code (which contains the variable that we want to edit during build time with ldflags) inside `internal` packages, as such are hidden from importing.

so, for instance have a file named `internal/constants/version.go`

```
package constants

var Version = ""
```

That we import in the code, and we later edit with ldflags

Reply:
@mudler Okay, so just to be clear, we want to store `var Version = ""` in this `version.go` file and set it on build using -ldflags. Any other logic that should be moved?

Reply:
I'll import the variable as the config is being set up and set it.

Reply:
yes basically it was good as it was the review before - just with slightly moving things in a different file ( inside an `internal` folder). `internal` is special for golang - as it allows local code to import it, but it doesn't allow remote code to import it.